awsshell/wizard.py

```diff
@@ -278,14 +292,15 @@ def _handle_retrieval(self):
             data = self._handle_static_retrieval()
         elif self.retrieval['Type'] == 'Request':
             data = self._handle_request_retrieval()
+        elif self.retrieval['Type'] == 'Wizard':
```

Comment:
I am fine with how you have it, but here is a cool way that we often use to avoid having to update an if statement every time you add a new type:

```py
retrieval_type = self.retrieval.get('Type', 'default').lower()
data = getattr(self, '_handle_' + retrieval_type + '_retrieval', self._handle_default_retrieval)()
```